### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by nmanumr**
Added few UI changes to header toolbar.

For Desktop:

image

For Mobile:

image

Mobile Search active:

image

### CODE REVIEW

**Comment by allanday** [comment]
I see vertical alignment issues in those screenshots - elements aren't centered. I'd also suggest sticking with all caps for the menu items.

Issue #54 has some relevant design work.

**Comment by nmanumr** [comment]
@allanday Are u talking about this loading spinner or the flathub icon??

**Comment by splitt3r** [comment]
It looks a lot better :). Awesome 😍

**Comment by allanday** [comment]
> @allanday Are u talking about this loading spinner or the flathub icon??

The Flathub icon is the most noticable issue - the vertical alignment is 3/4px off. The search box and its content also looks slightly off-center, to my eyes.

**Comment by nmanumr** [comment]
The feels a little off but it is not actually off. See here:

image

We have the spacing of 14px from top and 15px from bottom.

and search box have 1px shift with spacing of 13px top and 16px bottom.

**Comment by nmanumr** [comment]
I also changed toolbar links to uppercase

**Comment by nmanumr** [comment]
Fixed vertical alignment issue by cropping flathub icon

image

And also I noticed that there are two separate heights for both mobile and desktop (56px and 64px) in original flathub store, So I also fixed this one.

**Comment by splitt3r** [comment]
@nmanumr perhaps it would be nicer to use the https://github.com/angular/material2/blob/master/src/lib/toolbar/toolbar.scss#L4-L5 given variables for that.

**Comment by jgarciao** [comment]
Hi Nauman! Thanks for contributing to Fly and fixing the little issues
Allan remarked. I'll try your code next week

El ds., 27 d’oct. 2018, 11:57, Kai Schröer <notifications@github.com> va
escriure:

File: src/app/shared/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -1,75 +1,42 @@
...
+          <mat-icon class="search-box-icon hide-md">search</mat-icon>
+          <input #searchBox id="search-box" class="search-box" type="search" placeholder="Search apps" (keyup.enter)="onSearch(searchBox.value)" />
+        </div>
+      </div>
+    </div>
 
-    <mat-menu #menu="matMenu">
+    <div class="hide-md toolbar-nav">
+      <button mat-button routerLink="apps">APPLICATIONS</button>
+      <button mat-button href="https://github.com/flathub/flathub/wiki/App-Submission">PUBLISH</button>
```
Comment (jgarciao) [inline src/app/shared/toolbar/toolbar.component.html]: External links don't work with button. The code should be like this:

` <a mat-button href="https://github.com/flathub/flathub/wiki/App-Submission" aria-label="Publish an app on Flathub">PUBLISH</a>`

File: src/app/shared/toolbar/toolbar.component.html [hunk 2]
```diff
@@ -1,75 +1,42 @@
...
-    </a>
-
-    <div *ngIf="showSearchInput" class="search-box-container flex-fill-remaining-space">
-      <mat-icon class="search-box-icon">search</mat-icon>
-
-      <input #searchBox id="search-box" class="search-box-mobile" type="search" placeholder="Search apps" (keyup.enter)="onSearch(searchBox.value)"
-      />
+      <mat-menu #menu="matMenu" overlapTrigger="false">
+        <button mat-menu-item routerLink="apps">APPLICATIONS</button>
+        <button mat-menu-item href="https://github.com/flathub/flathub/wiki/App-Submission">PUBLISH</button>
```
Comment (jgarciao) [inline src/app/shared/toolbar/toolbar.component.html]: External links don't work with button. The code should be like this:

` <a mat-button href="https://github.com/flathub/flathub/wiki/App-Submission" aria-label="Publish an app on Flathub">PUBLISH</a>`

**Comment by jgarciao** [comment]
I've deployed the new version in the [test sever.](http://45.55.104.129) It looks good to me. 

@allanday  If you don't say anything otherwise I will merge with master and deploy it into production next Thursday 15th.

**Pull request merged**